WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/Add or Edit FIle/ProductDownloadFileViewModel.swift

```diff
@@ -0,0 +1,184 @@
+import Foundation
+import Yosemite
+
+/// Provides data needed for file settings.
```

Comment:
nit: We try to be consistent in the descriptions. I saw `downloads`, `downloadable files` and `files`. Let's try to use just `downloadable files`.
```suggestion
/// Provides data needed for downloadable file settings.
```

Reply:
uprated